projects/aca-content/src/lib/components/search/search-libraries-results/search-libraries-results.component.html

```diff
@@ -1,6 +1,7 @@
 <aca-page-layout>
   <aca-page-layout-header>
-    <adf-breadcrumb root="APP.BROWSE.SEARCH_LIBRARIES.TITLE"> </adf-breadcrumb>
```

Comment:
you are removing the title from the page, is that needed for the fix?

Reply:
I have done this so that it has the same look and feel as search results component. to keep the consistent look and feel.